File: swagger/models/get_staff_permissions_request.py
from .base_model import BaseModel


class GetStaffPermissionsRequest(BaseModel):
	"""
	no description available

	Attributes:

		staff_id
		The ID of the staff member whose permissions you want to return.
	"""

	input_map = {
		'StaffId': 'staff_id',
		}

	output_map = {
		'staff_id': 'StaffId',
		}

	staff_id = None



	def __init__(self, staff_id):
		self.staff_id = staff_id

File: swagger/Models/GetStaffPermissionsRequest.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetStaffPermissionsRequest extends BaseModel
{
	/**
	 * The ID of the staff member whose permissions you want to return.
	 * 
	 * @var integer format:int64
	 */
	public $staffId;
	/**
	 * Constructor
	 */
	public function __construct($staffId)
	{
		$this->staffId = $staffId;
	}

	protected function getInputMap()
	{
		return [
			'StaffId' => ['staffId', 'integer', 'int64'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'staffId' => ['StaffId', 'integer', 'int64'],
			];
	}
}
